### ISSUE TITLE
Zoom is not working

### ISSUE THREAD

**Issue by suprafly** [ayastreb/jekyll-maps#32]
I am trying to set the zoom on a map, but having no luck. Seems like it should be straight forward, but I've tried the following three iterations (separately, not all at once) without any success:

```{% google_map src="_data/google-maps" zoom="10" %}```

``` - title: "Paris"
   url: "http://google.fr"
   zoom: 10
   location:
     latitude: 48.8587741
     longitude: 2.2074741
```

``` - title: "Paris"
   url: "http://google.fr"
   location:
     latitude: 48.8587741
     longitude: 2.2074741
     zoom: 10
  ```

**Comment by ayastreb** [comment]
Hi,

yes, looks like something changed in GoogleMaps API and zoom doesn't work anymore :)
I'll take a look and try to fix it.

**Comment by suprafly** [comment]
You're amazing, thanks for the quick response. I forked it myself to poke around , so let me know if you need any help.

**Comment by suprafly** [comment]
It also might be useful to show in the readme the correct usage of the zoom option in yml, because it isn't clear if it should be nested under 'location' or not.

**Comment by ayastreb** [comment]
should be fixed now in 2.1.2

regarding usage: right now it's only possible to set zoom level as an attribute on `google_map` tag, e.g.
```
{% google_map  zoom="10" %}
```
see https://ayastreb.me/jekyll-maps/#examples

it is not possible to set zoom level from yml data-files.
